content/tutorials/grafana-fundamentals.md

```diff
@@ -345,6 +345,14 @@ Once you've acknowledged an alert, consider pausing it. This can be useful to av
 1. Find your alert in the list, and click the **Pause** icon on the right. The **Pause** icon turns into a **Play** icon.
 1. Click the **Play** icon to resume evaluation of your alert.
 
+### Clean up the local environment
+
+The tutorial has running Docker containers. When you want to clean up this local tutorial environment, run
```

Comment:
Still not telling me where to run this. Since the previous steps are in Grafana, we need to let the user know that they are shifting to a terminal of some sort.

Reply:
Unless you want me to revise the entire tutorial with this PR, this additional instruction follows what has been assumed in the earlier parts of the tutorial, where there were other docker-compose commands. My thinking:  if a customer already got the earlier tutorial steps working, they know how/where to run a Docker command.

Reply:
This is really fine, unless you are volunteering to edit the entire piece. This was intended as a simple edit. The intended audience knows how to use Docker. This isn't a total beginner-focused tutorial.